govcd/org.go

```diff
@@ -381,6 +381,101 @@ func (org *Org) GetTaskList() (*types.TasksList, error) {
 	return nil, fmt.Errorf("link not found")
 }
 
+// QueryAllOrgs returns all Org VDCs using query endpoint
+func (vcdclient *VCDClient) QueryAllOrgs() ([]*types.QueryResultOrgRecordType, error) {
+	filter := map[string]string{}
+
+	return vcdclient.Client.queryOrgList(filter)
```

Comment:
```suggestion
	return vcdclient.Client.queryOrgList(nil)
```

Reply:
suggestion will be applied in next commit.